pkgs/applications/misc/goldendict-ng/default.nix

```diff
@@ -31,13 +31,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "goldendict-ng";
-  version = "24.01.22";
+  version = "24.05.05";
 
   src = fetchFromGitHub {
     owner = "xiaoyifang";
     repo = "goldendict-ng";
-    rev = "v${finalAttrs.version}-LoongYear.3dddb3be";
-    hash = "sha256-+OiZEkhNV06fZXPXv9zDzgJS5M3isHlcOXee3p/ejpw=";
+    rev = "v${finalAttrs.version}-LiXia.ecd1138c";
+    hash = "sha256-C/0FUFLE3R+tZyCL88BiSFOHPTanILD/fIIQ/OQBSfk=";
```

Comment:
Are you sure of this hash? I had to change it to `sha256-XMsbI5qqDsunMLlzg8f5aaY+PY6NzO9UMN+Oy1Bt5ls=` on my end to make it work.

Reply:
Strange, I have to set the `sha256-XMsbI5qqDsunMLlzg8f5aaY+PY6NzO9UMN+Oy1Bt5ls=` on my  [23.11-based overlay](https://github.com/michojel/NixOS/blob/4b3a20b/overlays/goldendict-ng.nix#L14).
But `nixpkgs-review pr 309394` works with your hash and the same is produced by `nix-prefetch-git https://github.com/xiaoyifang/goldendict-ng --rev v24.05.05-LiXia.ecd1138c`.